### PR TITLE
Fix new page url with options

### PR DESCRIPTION
**Pull request by jkrems**
Comes down to a missing `return` that led to an undefined url.

### CODE REVIEW

File: lib/processes/proxy/proxy.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ function trimHash(url) {
 }
 
 function normalizeOptions(options) {
-  options.url = trimHash(options.url);
+  options.url = trimHash(options.url || '/');
```
Comment (khoomeister) [inline lib/processes/proxy/proxy.js]:
```
options.url = trimHash(options.url) || '/';
```
?

**Comment by khoomeister** [comment]
:shipit:

File: lib/processes/proxy/proxy.js (hunk 1 above)
Comment (jkrems) [inline lib/processes/proxy/proxy.js]: The problem is that `trimHash` will throw and kill the proxy if `options.url` is undefined. So the `||` has to be inside of the call.

Comment (khoomeister) [inline lib/processes/proxy/proxy.js]: Ahh OK gotcha

**Pull request merged**